### DISCUSSION TITLE
Where are the Client Access types? (Keycloak 19)

### DISCUSSION THREAD

**Discussion by joshlawreenz** [keycloak/keycloak#14157] [Q&A] [answered]
I've been searching Google and the Keycloak docs and I can't seem to find ANY info regarding this. I need set my client to Confidential but there is no option within the Clients. 

I'm currently using Keycloak 19. I'm half-sure that the option was present in Keycloak 18.
![image](https://user-images.githubusercontent.com/83667693/188057025-0694290d-3948-4556-9746-8b1fba5b98fb.png)

**Comment by darius-m** [comment, marked as answer]
The setting is called `Client authentication`, and it's in the screen capture you took. The tooltip mentions that when the setting is on, the client is set to confidential mode.

**Reply by joshlawreenz** [reply]
Solved! Thank you for answering!
